anatomy/tracks/concept-exercises.md

```diff
@@ -74,6 +80,8 @@ exercises
 
 **Purpose:** Introduce the concept(s) that the exercise teaches to the student.
 
+**Presence:** Required
```

Comment:
I've added this field to make it explicit if the field is required without having to consider any of the other text in this document.

Reply:
That's a great decision.